packages/perseus-editor/src/widgets/interactive-graph-editor/start-coords/util.ts

```diff
@@ -146,17 +147,13 @@ export const getQuadraticEquation = (startCoords: [Coord, Coord, Coord]) => {
     );
 };
 
-const findAngle = (point1: Coord, point2: Coord) => {
-    const x = point1[0] - point2[0];
-    const y = point1[1] - point2[1];
-
-    return (180 + (Math.atan2(-y, -x) * 180) / Math.PI + 360) % 360;
-};
-
-export const getAngleEquation = (startCoords: [Coord, Coord, Coord]) => {
-    const [point1, vertex, point2] = startCoords;
-
-    const angle = findAngle(point2, vertex) - findAngle(point1, vertex);
+export const getAngleEquation = (
+    startCoords: [Coord, Coord, Coord],
+    allowReflexAngles?: boolean,
+) => {
```

Comment:
another nit suggestion: adding the default param vs using an `||`, you can remove the declaration of `allowReflex`
```diff
- allowReflexAngles?: boolean,
+ allowReflexAngles: boolean = false,
....
- const allowReflex = allowReflexAngles || false;
- const angle = getClockwiseAngle(startCoords, allowReflex);
+ const angle = getClockwiseAngle(startCoords, allowReflexAngles);
```